src/webfaf/stats.py

```diff
@@ -50,10 +51,10 @@ def by_daterange(since, to):
     '''
 
     try:
-        if isinstance(since, str) or isinstance(since, unicode):
+        if isinstance(since, str) or isinstance(since, six.text_type):
             since = datetime.datetime.strptime(since, "%Y-%m-%d").date()
 
-        if isinstance(to, str) or isinstance(to, unicode):
+        if isinstance(to, str) or isinstance(to, six.text_type):
```

Comment:
And the same here. And you can get rid of that "or" condition as one `isinstance()` will be enough.